packages/process/process_monitor.pony

```diff
@@ -384,6 +399,17 @@ actor ProcessMonitor
         end
       | _stderr.near_fd =>
         _notifier.stderr(this, consume data)
+      | _err.near_fd =>
+        let child_errno: I32 = try data.read_u32(0)?.i32() else -1 end
```

Comment:
i think, if we don't use the errno here, we don't need to write it from the child.
Do you think it makes sense to keep it around?